styles/markers.atom-text-editor.less

```diff
@@ -0,0 +1,8 @@
+:host(:host) .gutter .line-number {
+  &.line-red {
+    background-color: red;
```

Comment:
We should not be picking colors directly. Use `@background-color-error` as a basis instead. My initial suggestion here would be to mimic the `highlight-*` classes as defined in the Atom styleguide. E.g.

```less
@import "ui-variables";
// ...
  &.latex-error {
    background-color: fade(@background-color-error, 50%);
    color: @text-color-error;
  }
// ...
```